omniauth-latvija.gemspec

```diff
@@ -2,22 +2,23 @@
 require File.expand_path('../lib/omniauth-latvija/version', __FILE__)
 
 Gem::Specification.new do |s|
-  s.name        = "omniauth-latvija"
+  s.name        = 'omniauth-latvija'
   s.version     = OmniAuth::Latvija::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Edgars Beigarts"]
-  s.email       = ["edgars.beigarts@makit.lv"]
-  s.summary     = "Latvija.lv authentication strategy for OmniAuth"
+  s.authors     = ['Edgars Beigarts']
+  s.email       = ['edgars.beigarts@makit.lv']
+  s.summary     = 'Latvija.lv authentication strategy for OmniAuth'
   s.description = s.summary
 
-  s.files         = Dir.glob("{lib}/**/*") + %w(README.md LICENSE)
-  s.require_paths = ["lib"]
+  s.files         = Dir.glob('{lib}/**/*') + %w(README.md LICENSE)
+  s.require_paths = ['lib']
 
-  s.add_runtime_dependency "omniauth", "~> 1.0"
-  s.add_runtime_dependency "canonix"
+  s.add_runtime_dependency 'omniauth', '~> 1.0'
+  s.add_runtime_dependency 'xmlenc'
```

Comment:
I don't see nokogiri here, c14n was added in nokogiri 1.5.1, so `s.add_runtime_dependency 'nokogiri', '>= 1.5.1'`

Reply:
Ah, Nokogiri is a dependency of `xmlenc`, that's why it's still working. WIll add it here